### DISCUSSION TITLE
azurefile-csi-driver outside of Azure datacenters?

### DISCUSSION THREAD

**Discussion by TomasKohout** [kubernetes-sigs/azurefile-csi-driver#935] [Q&A] [answered]
Hi guys,

I've a question whether or not it'd be possible to use Azure Files outside of Azure DCs. We have our own DCs and would like to use AzureFiles for some shared state/config etc. 

We would have to test it properly if latency is not too big etc., but I wasn't able to make it work so before I dive deeper I want to ask if it's even possible. :slightly_smiling_face: 

Thanks a lot and have a good time
Tomas

**Comment by andyzhangx** [comment, marked as answer]
It may work with smb protocol as long as `mount -t cifs ...` works between azure file server and your DC, follow guide here: https://github.com/kubernetes-sigs/azurefile-csi-driver/blob/master/README.md#option2-bring-your-own-storage-account-only-for-smb-protocol
